File: app/src/main/java/com/example/mapapp/MyClass/Vehicle.kt
package com.example.mapapp.ui.MyClass

data class Vehicle(
    // Prefixo do veículo
    val p: String,

    // Horário previsto para chegada do veículo no ponto de parada relacionado
    val t: String,

    // Indica se o veículo é (true) ou não (false) acessível para pessoas com deficiência
    val a: Boolean,

    // Indica o horário universal (UTC) em que a localização foi capturada.
    // Essa informação está no padrão ISO 8601
    val ta: String,

    //  Informação de latitude da localização do veículo
    val py: Double,

    // Informação de longitude da localização do veículo
    val px: Double
)

File: sp_movement/lib/app/modules/vehicle_position/models/domain/vehicle.dart
class Vehicle {
  int vehiclePrefix;
  bool isAccessibleForDisabilities;
  String timeLocationCaptured;
  double latitude;
  double longitude;

  Vehicle({
    required this.vehiclePrefix,
    required this.isAccessibleForDisabilities,
    required this.timeLocationCaptured,
    required this.latitude,
    required this.longitude,
  });
}
